### PR TITLE
add HostTemplate

### PR DESCRIPTION
**Pull request by jkahrs**
Support adding HostTemplates

### CODE REVIEW

File: host_template.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package zabbix
+
+type HostTemplate struct {
+	Templateid string `json:"templateid"`
```
Comment (AlekSi) [inline host_template.go]: Please rename to `TemplateId`.

**Comment by AlekSi** [comment]
May you please add test? You can use Vagrant with provided `Vagrantfile` to create a test setup.

**Comment by jkahrs** [comment]
One would need to add the CRUD for https://www.zabbix.com/documentation/2.2/manual/api/reference/template to make the tests actually work

**Comment by AlekSi** [comment]
See #24.

**Pull request closed without merging**